docs/aks.md

```diff
@@ -0,0 +1,177 @@
+with machine type of [Standard_B4ms](https://azureprice.net/vm/Standard_B4ms)
+and OS disk size reduced to 30 GiB. You can see detailed information about
+cluster creation options in the [AKS official documentation](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest).
+
+You may wait a few minutes for the cluster to be generated.
+
+Now you should configure the command-line access to your newly created cluster
+to make `kubectl` be able to use it.
+
+```bash 
+az aks get-credentials --resource-group my-resource-group --name my-cluster-name
+```
+
+# Install the Operator and deploy your MongoDB cluster
+
+1. Deploy the Operator. By default deployment will be done in the `default`
+    namespace. If that's not the desired one, you can create a new namespace
+    and/or set the context for the namespace as follows (replace the `<namespace name>` placeholder with some descriptive name):
+
+    ```bash
+    $ kubectl create namespace <namespace name>
+    $ kubectl config set-context $(kubectl config current-context) --namespace=<namespace name>
+    ```
+
+    At success, you will see the message that `namespace/<namespace name>` was created, and the context (`gke_<project name>_<zone location>_<cluster name>`) was modified.
+
+    Deploy the Operator using the following command:
+
+    ```bash
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/bundle.yaml
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbs.psmdb.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbbackups.psmdb.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbrestores.psmdb.percona.com serverside-applied
+        role.rbac.authorization.k8s.io/percona-server-mongodb-operator serverside-applied
+        serviceaccount/percona-server-mongodb-operator serverside-applied
+        rolebinding.rbac.authorization.k8s.io/service-account-percona-server-mongodb-operator serverside-applied
+        deployment.apps/percona-server-mongodb-operator serverside-applied
+        ```
+
+2. The operator has been started, and you can deploy your MongoDB cluster:
+
+    ```bash
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        perconaservermongodb.psmdb.percona.com/my-cluster-name created
+        ```
+
+    !!! note
+
+        This deploys default MongoDB cluster configuration, one mongos node and
+        one config server node. Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md#operator-custom-resource-options)
```

Comment:
@fiowro please check in cr.yml exact values.

Reply:
ouch :) done :)